addons/survey/controllers/survey_session_manage.py

```diff
@@ -169,7 +168,7 @@ def survey_start_short(self, session_code):
         This route is used in survey sessions where we need short links for people to type. """
 
         survey = self._fetch_from_session_code(session_code)
-        if survey:
+        if survey and survey.session_state in ['ready', 'in_progress']:
```

Comment:
What should happen in other cases ? Shouldn't we display something ? cc @dbeguin

Reply:
Hello @tde-banana-odoo 

In other cases, it will redirect the user to the session survey code page. Thanks